services/core-api/app/api/utils/helpers.py

```diff
@@ -29,3 +29,16 @@ def format_datetime_to_string(date):
 
 def format_currency(value):
     return '${:,.2f}'.format(float(value or 0)) if value else '$0.00'
+
+
+def get_preamble_text(description):
+    new_line = '\n\n'
+    return f"{description} for the {{mine_name}} {{application_type}} project was filed with the Chief Permitting Officer, " \
+           f"submitted on {{application_dated}} and last updated on {{application_last_updated_date}}.  The application included a " \
+           f"plan of the proposed work system (“Mine Plan”) and a program for the protection and reclamation of the surface of " \
```

Comment:
The variabled injected only have 1 curly brace surrounding  `{total_liability}`, should these only have 1?

Reply:
I see that is populated in the UI/db correctly...so disregard.

Reply:
Just formatting for string in python